content/enterprise_influxdb/v1.10/tools/influx_inspect.md

```diff
@@ -0,0 +1,587 @@
+---
+title: Influx Inspect disk utility
+description: >
+  Use the `influx_inspect` commands to manage InfluxDB disks and shards.
+menu:
+  enterprise_influxdb_1_10:
+    weight: 50
+    parent: Tools
+v2: /influxdb/v2.0/reference/cli/influxd/inspect/
+---
+
+Influx Inspect is an InfluxDB disk utility that can be used to:
+
+* View detailed information about disk shards.
+* Export data from a shard to [InfluxDB line protocol](/enterprise_influxdb/v1.10/concepts/glossary/#influxdb-line-protocol) that can be inserted back into the database.
+* Convert TSM index shards to TSI index shards.
+
+## `influx_inspect` utility
+
+### Syntax
+
+```
+influx_inspect [ [ command ] [ options ] ]
+```
+
+`-help` is the default command and prints syntax and usage information for the tool.
+
+### `influx_inspect` commands
+
+The `influx_inspect` commands are summarized here, with links to detailed information on each of the commands.
+
+* [`buildtsi`](#buildtsi): Converts in-memory (TSM-based) shards to TSI.
+* [`deletetsm`](#deletetsm): Bulk deletes a measurement from a raw TSM file.
+* [`dumptsi`](#dumptsi): Dumps low-level details about TSI files.
+* [`dumptsm`](#dumptsm): Dumps low-level details about TSM files.
+* [`dumptsmwal`](#dumptsmwal): Dump all data from a WAL file.  
+* [`export`](#export): Exports raw data from a shard in InfluxDB line protocol format.
+* [`report`](#report): Displays a shard level report.
+* [`reporttsi`](#reporttsi): Reports on cardinality for measurements and shards.
```

Comment:
Do we document the `report-disk` command?

Reply:
@lesam wrote it.

Reply:
We should have it documented - https://github.com/influxdata/docs-v2/issues/2112

Reply:
@davidby-influx weird - that's no longer there in the code, but there was a PR for it last year

Reply:
@lwandzura - what's no longer in the code?  

https://github.com/influxdata/influxdb/blob/1.10/cmd/influx_inspect/reportdisk/reportdisk.go

Reply:
it's not in our docs code
https://docs.influxdata.com/enterprise_influxdb/v1.9/tools/influx_inspect/

Reply:
I'll add back the report-disk after the v1.10 release docs are published - issue https://github.com/influxdata/docs-v2/issues/4440

Reply:
@lwandzura thank you for creating this issue https://github.com/influxdata/docs-v2/issues/4440

Reply:
@lwandzura think we can push out this Enterprise 1.10 content and loop back around on report-disk